contracts/api-spec.yaml

```diff
@@ -407,6 +407,37 @@ paths:
         '200':
           description: Successful deletion of the stream
 
+  /stream/statistics:
+    get:
+      operationId: retrieveStreamStats
+      description: Retrieve all the existing stream statistics
+      tags:
+        - Streams
+      responses:
+        '200':
+          description: Successful retrieval of stream statistics
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StreamStatsModel'
```

Comment:
I realized I misunderstood a question you asked me about this - we need an endpoint where we retrieve the stats of a single stream. We aren't really ever going to need to get all of the stream statistics (although we can leave it in if we want)
Something like GET /stream/1/statistics or /stream/statistics/1

Reply:
fixed